unitq integration (#1512)
DataDog/integrations-extras: unitq/README.md

# unitQ

## Overview

> The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. Datadog supports an advanced function API, which enables you to create graphs and alerts for user feedback metrics.

Reviewer: ```suggestion
The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog.By delivering unitQ metrics to Datadog, users can leverage Datadog's graphing and alerting capabilities to track user feedback ensuring a happy customer base
```

> The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. Datadog supports an advanced function API, which enables you to create graphs and alerts for user feedback metrics.

Author: We got a previous comment that said you don't allow marketing copy here. So I we previously had the copy describing unitQ but removed it due to a comment.

> The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. Datadog supports an advanced function API, which enables you to create graphs and alerts for user feedback metrics.

Reviewer 2: Hi @wahlforss,  just to clarify here - we recommend adding descriptive text about your product without any sales or marketing-forward language (such as phrases like "best in class", or "top rated"). I think adding a sentence here about what UnitQ does as a product would help highlight the value of the integration to possible new customers.

> The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. Datadog supports an advanced function API, which enables you to create graphs and alerts for user feedback metrics.

Author: I see. I'll add one sentence.

> The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog.By delivering unitQ metrics to Datadog, users can leverage Datadog's graphing and alerting capabilities to track user feedback ensuring a happy customer base

Reviewer 2: ```suggestion
The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. By delivering unitQ metrics to Datadog, users can leverage Datadog's graphing and alerting capabilities to track user feedback - ensuring a happy customer base.
```

## Setup

### Configuration

> 1. In unitQ, go to **Integrations**.

Reviewer 2: A hyperlink to the Integrations page within unitQ would be helpful here.

Author: Unfortunately, we can't do that at the moment.

# unitQ

## Overview

unitQ is your consolidated, searchable platform for user feedback. Our AI technology extracts data-driven insights from what users are saying to help you increase product quality.

> Our solution monitors millions of quality issues daily - for category-leading brands like Chime, Quizlet, Pandora, and Strava.

Reviewer: We try to avoid sales related material for the integrations page so I think we could remove this

Author: Done!

The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. Datadog supports an advanced function API, which enables you to create graphs and alerts for user feedback metrics.

## Setup

### Configuration

> 1. In unitQ, go to Integrations.

Reviewer 3: ```suggestion
1. In unitQ, go to **Integrations**.
```
